proteomic_aggregation_and_visualization/python/proteomic_aggregation_and_visualization.ipynb

```diff
@@ -0,0 +1,4004 @@
+{
```

Comment:
<p>Show these separately with a brief description of each or just choose one and describe a little more fully.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microbiomedata/nmdc_notebooks/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>